.github/workflows/e2e-v2-nydus.yml

```diff
@@ -18,7 +18,7 @@ env:
   KIND_VERSION: v0.12.0
   CONTAINERD_VERSION: v1.5.2
   NERDCTL_VER: 0.22.2
-  KIND_CONFIG_PATH: test/testdata/kind/config-v2.yaml
+  KIND_CONFIG_PATH: test/testdata/kind/config-v2-nydus.yaml
```

Comment:
Why create configuration?

Reply:
Removed now. 
At the beginning, the workflow was set up with two workers and one control plane, but now it's changed to one worker and one control plane, and it doesn't report insufficient disk space error when kind load image. I've changed back to using the same config.